### PR TITLE
New package: KyoboBook.eBook version 2.6.0.1

### PR DESCRIPTION
**Pull request by schembriaiden**
### Pull request has been created with [komac](https://github.com/russellbanks/Komac) v2.6.0 :rocket:

- Resolves #185371

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/185559)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978335226
> 2024-10-24 12:37:59.144 [CLI ] ShellExecute installer failed: 4294967293

Automated error analysis suggests -1978335226 may mean Running ShellExecute failed

(Automated response - build 897.)

**Comment by stephengillie** [comment]
> Automatic Validation ended with:
> 
> > Installation failed with exit code -1978335226
> > 2024-10-24 12:37:59.144 [CLI ] ShellExecute installer failed: 4294967293
> 
> Automated error analysis suggests -1978335226 may mean Running ShellExecute failed
> 
> (Automated response - build 897.)

Manual Validation ended with the same error. Running the installer manually caused another installer to run:
image
And eventually resulted in a completed install.

**Comment by vikingnope** [comment]
I tried to get it working, but it seems that the silent install is not working correctly

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978335226
> 2024-10-30 16:27:49.830 [CLI ] ShellExecute installer failed: 4294967293

Automated error analysis suggests -1978335226 may mean Running ShellExecute failed

(Automated response - build 897.)

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

File: manifests/k/KyoboBook/eBook/2.6.0.1/KyoboBook.eBook.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+- silentWithProgress
+InstallerSwitches:
+  Silent: /S
+  SilentWithProgress: /S
+UpgradeBehavior: install
+ReleaseDate: 2024-08-01
+Installers:
+- Architecture: x64
+  InstallerUrl: https://contents.kyobobook.co.kr/digital/download/elibrary/b2c/KyoboeBook_Setup.exe
+  InstallerSha256: BCB19157D755649BD9CB5B16C8060BB9BBE4B0B5AA904496F80C859EC183F55D
```
Comment (stephengillie) [inline manifests/k/KyoboBook/eBook/2.6.0.1/KyoboBook.eBook.installer.yaml:19]:
```suggestion
  InstallerSha256: 5410A20AC290FDA2206AA12C42AB7E02D5BADE20CB8CEEAEF9272D3832D793FB
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Manual Validation ended with:
> Actual hash:   5410A20AC290FDA2206AA12C42AB7E02D5BADE20CB8CEEAEF9272D3832D793FB
Expected hash: bcb19157d755649bd9cb5b16c8060bb9bbe4b0b5aa904496f80c859ec183f55d

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978335226
> 2024-11-05 05:41:49.402 [CLI ] ShellExecute installer failed: 4294967293

Automated error analysis suggests -1978335226 may mean Running ShellExecute failed

(Automated response - build 897.)

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by stephengillie** [comment]
Would it be possible to add the `Fassoo DRM Client` package to the repo also? If it could work as a dependency, then it might be possible to add silent install switches to that package. It's a long shot, but probably the best chance currently.

**Review by ItzLevvie — APPROVED** [review]

**Pull request closed without merging**